### PR TITLE
Added external links to doc it resources, roadmap, release notes to d…

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-3305

#### Changes proposed in this Pull Request:
-Added doc it links to roadmap, release notes and resources

#### Screenshots / Screen Recording (if applicable)
Screenshot 2023-03-16 at 1 25 47 PM

#### To Test:
- yarn start:reactdev

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by huayunh** [comment]
I don't know if this will be too much to ask for this PR. The new tag is intended to be shown every time there is a new release. As soon as the user clicked on that button, the new tag will be gone until the next release is out. Similar to how docit handles the announcement banner above the app bar.

**Comment by joebochill** [comment]
@JeffGreiner-eaton looks like we have an issue with the labeler

**Comment by joebochill** [comment]
> I don't know if this will be too much to ask for this PR. The new tag is intended to be shown every time there is a new release. As soon as the user clicked on that button, the new tag will be gone until the next release is out. Similar to how docit handles the announcement banner above the app bar.

Let's skip this for now. It was not explicitly called out in the acceptance criteria and it's a large amount of effort compared to how the story was estimated based on expectations. Let's make a new story for it to avoid scope creeping this.

**Comment by daileytj** [comment]
These buttons should be round...

It's easier to see when you click, but tough to grab a screenshot of my clicking without video sooo the hover screenshot will have to suffice for now
Screen Shot 2023-03-16 at 1 14 46 PM

**Comment by JeffGreiner-eaton** [comment]
> @JeffGreiner-eaton looks like we have an issue with the labeler

Yes I see. Multiple commits in same PR causing PR labeler to get removed. Not sure what to do about it.

**Comment by joebochill** [comment]
> > @JeffGreiner-eaton looks like we have an issue with the labeler
> 
> Yes I see. Multiple commits in same PR causing PR labeler to get removed. Not sure what to do about it.

Please create a JIRA story to investigate this. This is a very common use case that we need to work properly.

**Comment by surajeaton** [comment]
@daileytj Please review.

**Review by huayunh — CHANGES_REQUESTED** [review]
- [ ] You should not be using an icon button in this case. Do not force your users to click on a small button toward the end of your drawer item to navigate. The entire drawer item should be clickable; the external link thing should function only as a decoration, denoting that you are now taken to somewhere outside dev doc.
- [ ] You should not use `Color.gray[200]` in your code directly, as this (1) takes an extra step to import and (2) does not work in dark theme. Find the closest theme color and use it.
- [ ] Why is `titleDivider` set to `false`? Is that some specific design decision you made?
- [ ] Why are you modifying `drawer.tsx` instead of having your drawer navigation defined using `pageDefinitions` instead?

**Comment by huayunh** [comment]
- [ ] Why are these components in a folder on their own? If there is only one file other than index.tsx, it is not necessary to create a folder there.
image

**Comment by surajeaton** [comment]
> * [x]  Why are these components in a folder on their own? If there is only one file other than index.tsx, it is not necessary to create a folder there.
> 
> image

Removed ExternalLinkIcon component as we are not using it now.

**Comment by surajeaton** [comment]
> * [x]  You should not be using an icon button in this case. Do not force your users to click on a small button toward the end of your drawer item to navigate. The entire drawer item should be clickable; the external link thing should function only as a visual indicator, denoting that the users are now being taken to somewhere outside dev doc.
> * [x]  You should not use `Color.gray[200]` in your code directly, as this (1) takes an extra step to import and (2) does not work in dark theme. Find the closest theme color and use it.
> * [ ]  Why is `titleDivider` set to `false`? Is that some specific design decision you made?
> * [ ]  Why are you modifying `drawer.tsx` instead of having your drawer navigation defined using `pageDefinitions` instead?

In designs, I did not find the title divider. That's why I kept it false. Removed 
Screenshot 2023-03-21 at 12 01 28 PM

As discussed in the last week's bug scrub, it is ideal to add drawer nav group and nav item to drawer.tsx, we need to add icon (currently right comp) to the title. For that we need to alter the typings. But it is okay to directly add those in the drawer.tsx

File: docs/src/router/drawer.tsx [hunk 1]
```diff
@@ -136,6 +148,41 @@ export const NavigationDrawer: React.FC = () => {
                         items={convertNavItems(navGroup.pages || [], navGroup.path || '', 0, handleNavigate, dispatch)}
                     />
                 ))}
+                <DrawerNavGroup
+                    title="COMMUNITY"
+                    titleColor={Colors.blue[500]}
```
Comment (huayunh) [inline docs/src/router/drawer.tsx:153]: is it possible to put `"primary.main"` here instead of a direct color reference?

**Review by huayunh — COMMENTED** [review]
Behavior looks good

File: docs/src/router/drawer.tsx (hunk 1 above)
Comment (surajeaton) [reply docs/src/router/drawer.tsx]: We are not using sx prop in DrawerNavGroup instead we are using style, so passing a string will not parse. 
Screenshot 2023-03-23 at 1 47 29 PM

**Review by huayunh — APPROVED** [review]
very good!

**Pull request merged**